src/components/MoviesList/MoviesList.js

```diff
@@ -1,6 +1,21 @@
 import React from 'react';
+import { MovieCard } from '../MovieCard/MovieCard';
 import './MoviesList.scss';
 
-export const MoviesList = () => (
-  <>Put the list here</>
-);
+export const MovieList = ({ movies }) => {
+  const MoviesArray = movies.map(el => (
+    <div className="movies" key={el.imdbId}>
+      <MovieCard
+        title={el.title}
+        description={el.description}
+        imgUrl={el.imgUrl}
+        imdbUrl={el.imdbUrl}
+        imdbId={el.imdbId}
+      />
+    </div>
+  ));
```

Comment:
add this to the return statement, no need to create a separate var for it

Reply:
use spread operator to pass data
https://medium.com/@kevinyckim33/jsx-spread-operator-component-props-meaning-3c9bcadd2493

Reply:
add key to the MovieCard itself, div is extra

Reply:
propTypes?